editor/editor_node.cpp

```diff
@@ -656,10 +656,6 @@ void EditorNode::_notification(int p_what) {
 			editor_data.clear_edited_scenes();
 		} break;
 
-		case Control::NOTIFICATION_THEME_CHANGED: {
-			scene_tab_add_ph->set_custom_minimum_size(scene_tab_add->get_minimum_size());
-		} break;
-
```

Comment:
I don't think this was meant to be added in the first place... but now that `EditorNode` inherits Control this actually did get called and it caused a `nullptr` crash.
See https://github.com/godotengine/godot/pull/57086/files#r931703285